src/components/common/TokenIcon/styles.module.css

```diff
@@ -1,10 +1,12 @@
 .image {
   display: block;
   width: auto;
+  mix-blend-mode: multiply;
```

Comment:
As the background is orange in light mode this will make some logos appear slightly darker / more orange than they actually are:
![Screenshot 2023-04-26 at 12 31 44](https://user-images.githubusercontent.com/2670790/234549174-a10924fd-0d7d-48f7-9ed1-f458742828b4.png)
![Screenshot 2023-04-26 at 12 32 32](https://user-images.githubusercontent.com/2670790/234549379-2011b541-f4ff-42e1-8a44-3a2848d7bced.png)

Is there a specific reason why we want to multiply?

Reply:
To mask white background. I'll remove it though.